CONTRIBUTING.md

```diff
@@ -1,3 +1,5 @@
+<!-- markdownlint-disable no-inline-html no-duplicate-heading -->
```

Comment:
This line is referencing to a tool which is nowhere mentioned in the package.json or anywhere else. Do you think we can add the linting as a script to the package.json? Otherwise we can also remove that line.

Reply:
**Markdownlint** is an extension for **vscode**

https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

I can add it to `.devcontainer` and `.vscode/extensions.json`

Reply:
Done - #2400

Reply:
Well, markdownlint is more than just a vscode plugin: it's a ruby library, with a popular Node.JS port, the latter of which is used by markdownlint-cli2. There's also the aforementioned VSCode plugin for the Node CLI.
Point being, it's a large, well maintained project which you could add it to CI, etc, if desired.

Reply:
> Well, markdownlint is more than just a vscode plugin: it's a ruby library, with a popular Node.JS port, the latter of which is used by markdownlint-cli2. There's also the aforementioned VSCode plugin for the Node CLI. Point being, it's a large, well maintained project which you could add it to CI, etc, if desired.

Thanks for the information, it's a good point. However, from my point of view the installation of the extension (as done in #2400) is good enough for now. I don't want to enforce rules for markdown files because it's not too critical and important as the markdown files are not touched that often in this project. I only want to have biome.js as single tool of linting and formatting for now. Adding more logic to the CI and to maintain it, is not worth it from my point of view.